ui/src/utils/constants.ts

```diff
@@ -27,6 +27,7 @@ export const SCHEMA_TYPE = "Schema type";
 export const SCHEMAS = "Schemas";
 export const STATUS = "Status";
 export const VALUE_REQUIRED = "Value required";
+export const WARNING_ID = "warningNotification";
```

Comment:
This does not belong to this section. It's not a UI literal. I'd at least add a line break.

Reply:
And can be local because it's just used in one file

Reply:
The header for this section is "Literals used more than once" though. To me it makes sense to have it here in the absence of other structured ordering.

Reply:
This section is intended for literals readable by the user. Maybe we can improve the comment, but a local storage key is not visible to the user in the UI.

Reply:
You are right, I've added it to the file where it's used.